### PR TITLE
Add ICPC, Meetings, Contact pages

### PR DESCRIPTION
**Pull request by zhur123**
**Added**:

- OSU's ICPC History dating back to 1998 which pulls data from icpc.json in this repo
- Schedule of meetings of the most recent semester. It currently pulls content from the README of Spring 2022
    - https://raw.githubusercontent.com/cpcosu/Weekly-Events/main/Spring_2022/README.md
- Contact page with current and previous officers dating back to fall 2017 including officers that were apart of ACM Club
- Links page
    - image

**Future**: 
- Styling on the tables and text can always be better and we could use a js library for the tables in the future

**Resolved**: The long club name conflicted with the links in the nav bar so I decreased the padding and margins to make it fit on almost every screen.

**Issues I noticed**:

- After you scroll on a page, the active class disappears from the nav bar and it doesn't highlight the page you are currently on. Not sure where to look to fix this but not that big of a deal
- Nav bar is still not perfect as the bootstrap grid system makes only part of the club name clickable while the right side of the club name on the nav bar may not be clickable depending on the size of the screen and number of pages on the nav bar
- When the nav bar shrinks into a hamburger button on a small screen, the club name may be cut off and the pages are not centered which looks a bit weird
- Footer does not stick to the bottom of the Links page
- Subdomain www not registered for buckeyecode.club (FIXED by @robotgenis )

### CODE REVIEW

**Comment by zhur123** [comment]
Note: Add everyone's LinkedIn and the finished sponsorship packet

**Comment by zhur123** [comment]
Added LinkedIns, engineer contacts and sponsorship packet pdf.
Links page is also in this PR

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+                <p>Alex Li - Vice President</p>
+                <ul class="contact-info">
+                    <li>Email: <a href="mailto:li.8096@osu.edu">li.8096@osu.edu</a></li>
+                    <li>GitHub: <a href="https://github.com/Alex7Li">@Alex7Li</a></li>
+                </ul>
+            </div>
+            <div class="col-sm-4 col-md-4 text-center officer">
+                <p>Dingkang Wang - Treasurer</p>
+                <ul class="contact-info">
+                    <li>Email: <a href="mailto:wang.6150@osu.edu">wang.6150@osu.edu</a></li>
```
Comment (hczhcz) [inline contact.html:318]: GitHub: https://github.com/wangdingkang

Comment (zhur123) [reply contact.html:318]: Added to master

**Pull request merged**